frontends/web/tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        // https://github.com/palantir/tslint/blob/master/src/configs/recommended.ts
+        "tslint:recommended"
+    ],
+    "jsRules": {},
+    // https://palantir.github.io/tslint/rules/
+    "rules": {
+        "arrow-parens": [true, "ban-single-arg-parens"],
+        "interface-name": [true, "never-prefix"],
+        "max-line-length": [false],
+        "member-ordering": [false],
+        "object-literal-sort-keys": [false],
+        "quotemark": [true, "single", "jsx-double", "avoid-escape"]
+    },
+    "rulesDirectory": []
+}
```

Comment:
having comments in .json files still weirds me out but I can live with it if you insist